### ISSUE TITLE
Idea: Compilation to TASTY

### ISSUE THREAD

**Issue by magnus-madsen** [flix/flix#318]
A longterm goal, once the JVM backend is complete for both the functional and logic languages, would be to implement support for the TASTY intermediate language of Dotty.

A TASTY backend, if possible, would provide for easy interop with Scala in the future.

However, this project should probably not be undertaken before TASTY has been finalized and Dotty has seen some adoption (and can compile Flix). Furthermore, the JVM backend ought to be complete so that it can be used as a reference point.

**Comment by magnus-madsen** [comment]
Postponed indefinitely.
